pom.xml

```diff
@@ -198,6 +198,12 @@
             <artifactId>javax.el-api</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+            <version>1.7.7</version>
+            <scope>compile</scope>
```

Comment:
we should add `<optional>true</optional>` here, in case a user doesn't want to use SLF4J logging, this dependency won't be in his project